packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -765,7 +765,17 @@ export const audiusBackend = ({
     userTagCount,
```

Comment:
Wdyt about moving `searchTags` into `AudiusAPIClient` with regular search? Then we wouldn't need libs changes, and it would be more consistent

Reply:
Yea I think that would be awesome. I can make a ticket for it